Test/com/nicoles_change_maker/test/ChangeMakerTest.java

```diff
@@ -0,0 +1,235 @@
+package com.nicoles_change_maker.test;
+import com.nicoles_change_maker.ChangeMaker;
+import org.junit.*;
```

Comment:
What are you actually using from JUnit? (I think it's actually just one class and one static method). It's a better practice to import each dependency explicitly by name: it gives readers a much clearer idea of how many dependencies this class actually has.